<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 242, 246);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="left_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>374</width>
       <height>425</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>374</width>
       <height>425</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="inner_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>290</width>
          <height>277</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border: 1px solid #3085FE;
border-radius:10px;}
QPushButton{
border-radius:10px;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhUrlCharactersOnly</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="id_lab">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="id_lineedit">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pw_lab">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pw_lineedit">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="login_btn">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(238, 239, 240);</string>
           </property>
           <property name="text">
            <string>로그인하기</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="face_check_btn">
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(48, 133, 254);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>얼굴인식</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="right_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>596</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>596</width>
       <height>425</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>375</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="face_lab">
           <property name="minimumSize">
            <size>
             <width>356</width>
             <height>356</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>356</width>
             <height>356</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../img/icon/resource.qrc">:/face-id.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="explain_lab">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>얼굴인식 버튼을 클릭해 주세요!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../img/icon/resource.qrc"/>
  <include location="../../img/icon/resource.qrc"/>
 </resources>
 <connections/>
</ui>
